### ISSUE TITLE
Docking, Boat too long

### ISSUE THREAD

**Issue by claudejdev** [PygmalionOfCyprus/cmo-db-requests#4344]
### DB Selector

Both

### Affected DBID(s)

Ship 3615 "It won't fit"

### Description of Problem

An LCU cannot be rebased into its mothership, as its size would be "too long" as per the error that is prompting.
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/35732829/d8628910-77fa-4b0d-bcf2-7a8bd243b67f)

However, it should. If it is determined against the size and docking facilities parameters.

LCU
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/35732829/b059aa0c-b997-417c-b853-18f92bea97bb)

Motherships
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/35732829/b45c6494-8641-4e48-99e3-64ebac123bfa)
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/35732829/6e327a69-4eea-4205-a0ff-c887ecadee4a)

### Recommended Fix

TLDR; insert a new Docking PhysicalSize "Very Large Dock (LCU/LCAC, 45.1-200 m Long)", to allow large Landing Craft Utility (LCU) to be docked properly in an LPD.

---

Having that new size available would allow simulating the operation of ships such as the French EDIC/CDIC class LCU aboard the Foudre-class landing platform dock, and possibly other Cold War era LCU, that operated until the 2000s'.

And while I would like to see it included in both DBs that we enjoy now, it could also make its way to the WW2 DB, if that ever finds a purpose.

I'm failing to see any downside to doing it, but the time spent.

![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/35732829/99d7dfa5-057a-4f0b-8446-7e5dddc321ce)
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/35732829/6dba63df-6a0b-4268-9c99-caf2b398d3a7)

[[1328.12] Docking, Boat too long - Matrix Games Forums » Thu Feb 22, 2024 7:17 am](https://www.matrixgames.com/forums/viewtopic.php?f=10230&t=400409&p=5151875#p5151875)
[Suggestion : new Docking PhysicalSize to fit large LCU inside LPD - Matrix Games Forums » Tue Mar 29, 2022 6:49 pm](https://www.matrixgames.com/forums/viewtopic.php?f=10230&t=382097)
[France - LSD L 9021 Ouragan (Deep Review) · Issue #854 · PygmalionOfCyprus/cmo-db-requests >> Apr 3, 2022](https://github.com/PygmalionOfCyprus/cmo-db-requests/issues/854)
